pkgs/development/python-modules/dendropy/default.nix

```diff
@@ -4,42 +4,39 @@
   fetchFromGitHub,
   pytestCheckHook,
   pythonOlder,
+  nix-update-script,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "dendropy";
-  version = "4.5.1";
-  format = "setuptools";
+  version = "5.0.1";
 
-  disabled = pythonOlder "3.7";
+  pyproject = true;
+  build-system = [ setuptools ];
 
   src = fetchFromGitHub {
     owner = "jeetsukumaran";
     repo = pname;
-    rev = "v${version}";
-    hash = "sha256-FP0+fJkkFtSysPxoHXjyMgF8pPin7aRyzmHe9bH8LlM=";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-4VkFkY4/hDfb6elOSTDia6tRJm73c1YP9nBW2dJsxsI=";
   };
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  postPatch = ''
+    substituteInPlace setup.py --replace '["pytest-runner"],' '[],'
+    substituteInPlace src/dendropy/interop/paup.py --replace '"paup"' '"NONE"'
```

Comment:
This is a magic string that disables PAUP integration. PAUP is not easily downstramable into nixpkgs.

Reply:
How about setting `env.DENDROPY_PAUP_EXECUTABLE_PATH = "NONE"` instead?
I think it is more stable than patching.

Reply:
> How about setting env.DENDROPY_PAUP_EXECUTABLE_PATH = "NONE" instead?

Done